lib/rust/ensogl/core/src/debug/monitor.rs

```diff
@@ -931,3 +807,14 @@ stats_sampler!(
     0,
     1.0
 );
+
+/// Keeps the body if the `statistics` compilation flag was enabled.
+#[macro_export]
+macro_rules! if_compiled_with_stats {
+    ($($tok:tt)*) => {
+        #[cfg(feature = "statistics")]
+        {$($tok)*}
+        #[cfg(not(feature = "statistics"))]
+        {}
+    };
+}
```

Comment:
Does the profiling framework work if the `statistics` flag is off?

Reply:
As discussed, I added a note in the docs, mentioning that some stats will be zero in such case (specifically, the note is in the `stats.rs` module docs).